File: app/src/main/java/ru/coffeeplanter/androidlogin/platform/ResourceSupplier.java
package ru.coffeeplanter.androidlogin.platform;

/**
 * Created by Ilya Solovyov on 18.09.2017.
 * is3k@ya.ru
 */

public class ResourceSupplier {

    public static String getString(int stringId) {
        return App.getContext().getString(stringId);
    }

}

File: app/src/main/java/ru/coffeeplanter/androidlogin/platform/ResourceSupplier.kt
package ru.coffeeplanter.androidlogin.platform

/**
 * Created by Ilya Solovyov on 18.09.2017.
 * is3k@ya.ru
 */

object ResourceSupplier {

    fun getString(stringId: Int): String {
        return App.context?.getString(stringId) ?: ""
    }

}
